google/resource_compute_image.go

```diff
@@ -195,3 +213,9 @@ func resourceComputeImageDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceComputeImageImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	d.Set("create_timeout", computeImageCreateTimeoutDefault)
```

Comment:
Can you explain why this is necessary instead of setting `create_timeout` on read every time?